docs/en/admins/02_Installation.md

```diff
@@ -141,4 +141,6 @@ A step-by-step tutorial is available [in French](http://www.pihomeserver.fr/2013
 
 # Security
 
+Make sure to expose only the `./p/` folder on the web, the other directories contain personal and sensitive data. 
```

Comment:
There are examples for this somewhere in common configurations (Apache, nginx). I think they should either be moved here or linked to.

Reply:
Yes in the same file, only a couple of lines above this sentence

Reply:
I see. Maybe it'd be better to add a pointer? You might still have a somewhat similar tunnel vision by just using the TOC, although not as bad as GH literally hiding most of the file. :-P

Reply:
Hm, I am not sure if I get your point. 
I'll add another sentence "See the Apache and nginx config examples above.", or did you mean something different?

Reply:
Yes, that should do.